journalwatch.py

```diff
@@ -364,15 +369,23 @@ def parse_config_files():
     return cfg, patterns
 
 
-def get_journal(since=None):
+def get_journal(since=None, priority=None):
     """Open the journal and get a journal reader.
 
     Args:
         since: A datetime object where to start reading.
                If None, the whole journal is read.
+        priority: A number between 1 and 7 for lowest priority
+               to consider.
+               If omited, defaults to "LOG_INFO", number 6.
     """
     j = journal.Reader()
-    j.log_level(journal.LOG_INFO)
+    if (int(priority) < 1):
```

Comment:
Just do `priority = int(priority)` once at the top. Also, you should catch `ValueError` (for invalid integers) and raise a `JournalWatchError` instead.